src/BasicViewer.js

```diff
@@ -33,17 +35,32 @@ var BasicViewer = function(catTitle, features, container, fv) {
                         return 'up_pftv_feature up_pftv_' + d.type.toLowerCase();
                     })
                 ;
+                newShapes
```

Comment:
Could you check, as "newShapes" is on enter() that they get removed on the exit contect? "shapes" are taken care of by shapes.exit().remove()